recipes/nomkl/run_test.py

```diff
@@ -0,0 +1,7 @@
+import subprocess
+import json
+
+args = ("conda", "list", "mkl", "--json")
```

Comment:
What is this supposed to do?

Reply:
Check if `mkl` is in the env. We don't really need that but I could not come up with a better test. I'm open to suggestions.